packages/playground/data-liberation/src/import/WP_Entity_Importer.php

```diff
@@ -108,6 +113,9 @@ public function __construct( $options = array() ) {
 				'default_author'            => null,
 			)
 		);
+
+		WP_Topological_Sorter::activate();
```

Comment:
This implicitly runs a bunch of database operations in the object constructor. If we create 10 objects, it will run 10 times. If it fails, we don't have a good way of reporting that. What do you think about running it in a plugin-related hook instead?

Reply:
It has already run for the first time during the activations. I can remove it from here.